CONTRIBUTING.md

```diff
@@ -160,6 +160,44 @@ const MyComponent: FC<MyComponentProps> = ({ prop1, prop2... }) => (
 export default MyComponent;
 ```
 
+#### How to I18n my component
+
+In Node.js website we use [react-intl](https://formatjs.io/docs/react-intl/) to handle internationalization.
+
+There are two types of components, one uses content write in markdown and the other uses react-intl formatted message.
+
+- Using markdown:
+
+```tsx
+// ... your code
+
+const Component: FC<Props> = ({ children }) => (
+  <div>
+    <h1>{children}</h1>
+  </div>
+);
+```
+
+- Using react-intl:
+
+```tsx
+import { FormattedMessage } from 'react-intl';
+
+// ... your code
+
+const Component: FC<Props> = ({ children }) => (
+  <div>
+    <FormattedMessage id="your-message-id" />
+  </div>
+);
+```
+
+How to add a new message:
```

Comment:
This is repeated instructions from the TRANSLATION guidelines to a certain degree. Please updated the TRANSLATION.md instead of CONTRIBUTING.md